jest.config.base.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     '^.+\\.(t|j)sx?$': [
       '@swc/jest',
       {
-        sourceMaps: true,
+        sourceMaps: 'inline',
```

Comment:
This is the jest config for the tests run against the create-plugin and sign-plugin source code rather than the jest config that is scaffolded for a plugin. Rather than make the change here could you please make it in [packages/create-plugin/templates/common/.config/jest.config.js](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/common/.config/jest.config.js#L28) instead?

Reply:
Yeap. It was the wrong file.
Now I hope the changes are in the right file